exo.jcr.ext.services/src/main/resources/db/changelog/jcr-index.db.changelog-1.0.0.xml

```diff
@@ -53,4 +53,9 @@
       <append value=" ENGINE=INNODB CHARSET=UTF8 COLLATE utf8_general_ci"/>
     </modifySql>
   </changeSet>
+  <changeSet author="jcr-index" id="1.0.0-4" dbms="oracle,postgresql">
```

Comment:
```suggestion
  <changeSet author="jcr-index" id="1.0.0-4" dbms="postgresql">
```

Reply:
I keep using oracle here because it's known that the same error will be raised in Oracle as well (even that we don't support it anymore, just in case ;-) )